### ISSUE TITLE
deploy: operatator.yaml -> "exec: \"yugabyte-operator\": executable file not found 

### ISSUE THREAD

**Issue by mcuadros** [yugabyte/yugabyte-operator#12]
The default `Deployment` describe in `deploy/operator.yaml`, try to execute the command `yugabyte-oprtator`.

This doesn't work:
```
OCI runtime create failed: container_linux.go:345: starting container process caused "exec: \"yugabyte-operator\": executable file not found in $PATH":  unknown

```

**Comment by bhavin192** [comment]
Hey @mcuadros, thank you for reporting this issue and proposing a fix, appreciate your willingness to help! The image tagged with `:latest` on [quay.io/repository/yugabyte/yugabytedb-operator](https://quay.io/repository/yugabyte/yugabytedb-operator?tag=latest&tab=tags) was not update according to new changes. It has been fixed now, can you please try to deploy the operator again or just restart the deployment?

```
kubectl rollout restart deployment yugabyte-operator -nyb-operator
kubectl rollout status deployment yugabyte-operator -nyb-operator
```

Basically the old image had a binary with name `yugabyte-k8s-operator` instead of `yugabyte-operator`.


Edit: sorry for similar comment as of one on the PR.

**Comment by mcuadros** [comment]
https://github.com/yugabyte/yugabyte-operator/pull/13#issuecomment-618208106
